### DISCUSSION TITLE
[Other] Is it possible to have a container image with rabbitmq_stream and rabbitmq_stream_management plugins enabled?

### DISCUSSION THREAD

**Discussion by paulomorgado** [rabbitmq/rabbitmq-server#13004] [Other] [answered]
### Community Support Policy

- [X] I have read [RabbitMQ's Community Support Policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)

### RabbitMQ version used

4.0.3

### How is RabbitMQ deployed?

Community Docker image

### Steps to reproduce the behavior in question

I'd like to add that as an [Aspire](https://aka.ms/dotnet/aspire) resource to my development.

**Comment by lukebakken** [comment, marked as answer]
Hi @paulomorgado 

Team RabbitMQ are not the primary maintainers of the Docker image, though @michaelklishin and I contribute to it. I suggest asking your question here:

https://github.com/docker-library/rabbitmq/discussions

For what it's worth, it's easy to accomplish enabling those plugins via your own image:

https://github.com/lukebakken/rabbitmq-server-13004

**Comment by michaelklishin** [comment]
Attach an [`enabled_plugins` file](https://www.rabbitmq.com/docs/plugins#enabled-plugins-file) to your image and you can enable any combination of plugins.
